src/integrations/optimizely.ts

```diff
@@ -366,3 +366,8 @@ export const getEnabledFeatures = () => {
     audienceSegmentationAttributes
   )
 }
+
+/**
+ * Export imported types
+ */
+export {ToggleIdType, VariantType}
```

Comment:
For sake of consistency I think we can release this as 4.0.3.

Reply:
Will do. Just want to have the bot set up first before adding a changeset just to ensure that it works correctly :)